cogito/ghcommitsink.go

```diff
@@ -93,9 +93,21 @@ func ghMakeContext(request PutRequest) string {
 }
 
 func apiEndpoint(h string) string {
-	hostname := strings.ToLower(h)
+	hostname := normalizeHostname(h) // not really required; already called during protocol init
+	if hostname == github.GhDefaultHostname {
+		return github.API
+	}
 	if strings.Contains(hostname, "127.0.0.1") {
 		return fmt.Sprintf("http://%s", hostname)
 	}
-	return github.API
+	return fmt.Sprintf("https://%s/api/v3", hostname)
```

Comment:
supports the GHEnterprise hostnames...

Reply:
Personally, I find all the logic in: https://github.com/cli/cli/blob/trunk/internal/ghinstance/host.go#L24 
inverted for my taste..

Reply:
The logic here is indeed direct. As long as it works, I agree it is simpler to understand, good!